zanata-war/src/main/webapp/WEB-INF/pages.xml

```diff
@@ -24,15 +24,10 @@
   </page>
 
 
-  <page view-id="/dashboard/*" login-required="true">
-    <restrict>#{identity.loggedIn}</restrict>
-  </page>
-
   <page view-id="/edit_home_content.xhtml">
     <action execute="#{breadcrumbs.clear}" />
     <action
       execute="#{breadcrumbs.addLocation('', msgs['jsf.server.EditHomePage.label'])}" />
-    <restrict>#{s:hasRole('admin')}</restrict>
```

Comment:
Where did this check move to?  I can see `@CheckRole("admin")` on ServerConfigurationBean; is there anything else?